### PR TITLE
fix: runSchedule get filePath keep same logic with loader

### PR DESCRIPTION
**Pull request by mansonchor**
for support runtime dir source file symlink

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -74,9 +79,14 @@ module.exports = app => {
     if (path.isAbsolute(schedulePath)) {
       schedulePath = require.resolve(schedulePath);
     } else {
+      const ext = (process.env.EGG_TYPESCRIPT === 'true' &&  Module._extensions['.ts'])
```
Comment (atian25) [inline app.js:82]: require.extensions 就可以了吧

File: app.js [hunk 2]
```diff
@@ -74,9 +79,14 @@ module.exports = app => {
...
       schedulePath = require.resolve(schedulePath);
     } else {
+      const ext = (process.env.EGG_TYPESCRIPT === 'true' && Module._extensions['.ts'])
+        ? '.ts'
+        : '.js';
       for (const dir of directory) {
+        const findPath = path.join(dir, schedulePath + ext);
         try {
-          schedulePath = require.resolve(path.join(dir, schedulePath));
+          require(findPath);
```
Comment (atian25) [inline app.js:88]: 为啥要 require？我记得文件不存在时，require.resolve 就会抛错了

Comment (mansonchor) [reply app.js:88]: 其实这里都不太想 require.resolve 了，保持和 loader 一样的逻辑去拼这个完整的 filePath 就好

Comment (atian25) [reply app.js:88]: require 的里面其实就是 require.resolve

Comment (mansonchor) [reply app.js:88]: 我意思是这里不需要 try catch 去 require 一下了，这个完整路径是可以直接拼出来的

**Review by atian25 — APPROVED** [review]

**Review by killagu — APPROVED** [review]
LGTM

**Pull request merged**